native/lambda_game_engine/src/game.rs

```diff
@@ -175,6 +175,40 @@ impl GameState {
         }
     }
 
+    fn activate_skills(&mut self) {
```

Comment:
can't we reuse `activate_skill` for this function?

Reply:
I couldn't find a way to reuse that function. That function is called from the user's input (client) and I don't have some parameters. However, I agree we could do a refactor to reutilize some code of it. We can do it in a future PR. What do you think?

Reply:
same here, it's better to do the refactor afterwards since we need this to implement uma